### DISCUSSION TITLE
Get GitHub issues / project data in jira

### DISCUSSION THREAD

**Discussion by ScopeSV** [atlassian/github-for-jira#2481] [Q&A]
Hi!
Our organization's management uses Jira, but the engineers use Github projects and issues to keep track of development.

Is there a way to track issues and projects in Jira, or are only PRs and commits supported?

BR
Stephan Valois

**Comment by everforyou** [comment]
Hi,

To integrate GitHub with Jira and track issues and projects, follow these steps:

**Install the GitHub for Jira App**:
   - Go to the Atlassian Marketplace.
   - Search for "GitHub for Jira" and install the app.

**Connect Your GitHub Account**:
   - In Jira, navigate to **Jira settings** > **Apps** > **GitHub**.
   - Click on **Connect GitHub account** and follow the prompts to authorize Jira with your GitHub account.

**Link Repositories**:
   - Once connected, you can link specific GitHub repositories to your Jira project.
   - This allows Jira to pull in information about commits, pull requests, and issues.

**Use Smart Commits**:
   - In your commit messages, use the format `JIRA-123 #comment Your comment` to link commits to Jira issues. Replace `JIRA-123` with your actual issue key.
